tags/src/lib.rs

```diff
@@ -255,7 +255,7 @@ impl TagsContext {
         config: &'a TagsConfiguration,
         source: &'a [u8],
         cancellation_flag: Option<&'a AtomicUsize>,
-    ) -> Result<impl Iterator<Item = Result<Tag, Error>> + 'a, Error> {
+    ) -> Result<(impl Iterator<Item = Result<Tag, Error>> + 'a, bool), Error> {
```

Comment:
I wanted to turn this into a proper type rather than a Result over a boolean and iterator of Results, but we’d need to opt into some nightly features to implement the `Try` trait, or I’d have needed to replumb a ton of code, so I just did the easy fix and made it a tuple.

Reply:
I think the tuple is fine.